### PR TITLE
Feature/712 documentation overhaul

### LINKED ISSUE #712: Documentation Overhaul
## Description
Currently, without onboarding, getting Lodestar up & running will encounter some issues, a refactor of the documentation in both structure and approach should be undertaken to improve the developer experience of the repo

# Objectives

- [ ] Ensure directory context is uniform between package readmes
- [ ] Tl;Dr summary
- [ ] Prerequisite/Dependencies section
- [ ] Create a Getting started section
- [ ] Expand on code examples & testing suites
- [ ] Expand on context where possible

### PR DESCRIPTION
**Pull request by RyRy79261**
closes #712 

Documentation changes will be smaller, more iterative amendments thus merging this initial overhaul now

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +11,70 @@
...
+* [Prerequisites](#prerequisites)
+* [Getting Started](#getting-started)
+* [Contributors](#contributors)
+* [Meeting Notes](#meeting-notes)
+* [Donations](#donations)
+* [Packages](#packages)
+
+## Tl;Dr:
+
+Lodestart is a Typescript implementation of the Eth 2 Beacon chain spec.
```
Comment (wemeetagain) [inline README.md]: Typo here.

File: packages/lodestar/README.md [hunk 2]
```diff
@@ -3,42 +3,51 @@
...
+* [Yarn](https://yarnpkg.com/)
 
-` ./bin/lodestar dev -v 8 -p minimal`
 
-#### Using start-multi.sh
 
-`sh ./start-multi.sh <number_of_beacon_nodes> <total_validators> <validator_distribution_per_node>`
-`sh ./start-multi.sh 4 8 8,0,0,0`
+## What you need
+You will need to go over the [specification](https://github.com/ethereum/lodestar-specs). You will also need to have a [basic understanding of sharding](https://github.com/ethereum/wiki/wiki/Sharding-FAQs). Note that that the specification is an ongoing document and will get outdated. The reference implementation by the Ethereum development team is written in Python and can be found [here](https://github.com/ethereum/beacon_chain).
```
Comment (wemeetagain) [inline packages/lodestar/README.md]: I think this paragraph is almost entirely stale (I see it was just moved). Might think about removing entirely, or only leaving the first sentence. The link for the spec is also https://github.com/ethereum/eth2.0-specs

**Review by wemeetagain — CHANGES_REQUESTED** [review]

File: packages/lodestar/README.md [hunk 3]
```diff
@@ -3,42 +3,51 @@
...
 You can always download spec test cases by running `yarn install --force`.
 
+## Run dev
+
+` ./bin/lodestar dev -v 8 -p minimal`
+
+#### Using start-multi.sh
+
+`sh ./start-multi.sh <number_of_beacon_nodes> <total_validators> <validator_distribution_per_node>`
+`sh ./start-multi.sh 4 8 8,0,0,0`
```
Comment (mpetrunic) [inline packages/lodestar/README.md]: I don't think this works anymore. I think I've commented it out on master

**Review by mpetrunic — COMMENTED** [review]
I very much like this :smile:

File: packages/lodestar/README.md (hunk 2 above)
Comment (RyRy79261) [reply packages/lodestar/README.md]: Roger that, will amend

File: packages/lodestar/README.md (hunk 3 above)
Comment (RyRy79261) [reply packages/lodestar/README.md]: Ok cool will comment out, although is there something to the same effect to put in?

Comment (wemeetagain) [reply packages/lodestar/README.md]: Not currently

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**